SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -167,6 +167,18 @@ bool USpatialActorChannel::IsDynamicArrayHandle(UObject* Object, uint16 Handle)
 	return RepLayout->Cmds[RepLayout->BaseHandleToCmdIndex[Handle - 1].CmdIndex].Type == ERepLayoutCmdType::DynamicArray;
 }
 
+void USpatialActorChannel::UpdateShadowData()
+{
+	FObjectReplicator& ActorObjReplicator = FindOrCreateReplicator(Actor).Get();
```

Comment:
Actor's aren't always valid in SpatialActorChannel, worth verifying the Actor pointer is valid and it isn't pending kill.

Reply:
We do know it's valid though because the only codepath that leads here already check Actor validity.
Still, a `check(Actor);` as a sanity check shouldn't hurt. (or `checkSlow(Actor);`)